src/helper/streaming/sse.ts

```diff
@@ -42,10 +42,24 @@ const setSSEHeaders = (context: Context) => {
   context.header('Connection', 'keep-alive')
 }
 
-export const streamSSE = (c: Context, cb: (stream: SSEStreamingApi) => Promise<void>) => {
+export const streamSSE = (
+  c: Context,
+  cb: (stream: SSEStreamingApi) => Promise<void>,
+  onError?: (e: Error, stream: StreamingApi) => Promise<void>
+) => {
   const { readable, writable } = new TransformStream()
   const stream = new SSEStreamingApi(writable, readable)
-  cb(stream).finally(() => stream.close())
+  ;(async () => {
+    try {
+      await cb(stream)
+    } catch (e) {
+      if (e instanceof Error && onError) {
+        await onError(e, stream)
+      }
```

Comment:
If `onError` was not set, how about outputting an error? This way the user may not know that an error has occurred.